utils/auto_dockerfile_update/get_dockerfiles.py

```diff
@@ -90,7 +91,34 @@ def filter_ignored_files(files_list):
         print(f'could not read ignored config {str(e)}')
         return files_list
 
-
+def get_file_path_and_docker_version_if_exist(dockerfile: dict, latest_tag:str)-> Tuple:
```

Comment:
i would return an Optional[Tuple] instead of having bool be the last return value, but matter of choice